content/monetization/app-review-process/index.mdx

```diff
@@ -64,105 +62,118 @@ Don't inlcude app description in the title - there's plenty of space for that in
+The App Review process has 4 steps.
+Developers have constant access to the current status of the App Review process in **Developer Console > App > Monitor > Overview**.
 
-- **Developing** - App is during the development process. Developer is able to make all the changes.
+- **Developing** - Your app is in development. As a Developer, you're able to work on the app and make changes to it.
 
-- **Submitted** - App has been submitted to the review. Developer can't make any changes to the app, unless they decide to cancel submission.
+- **Submitted** - The app has been submitted for review. You can no longer make changes to the app unless you decide to cancel the submission.
 
-- **Accepted** - App has been accepted. Developer can publish the app to the public Marketplace.
+- **Accepted** - The app has been accepted. You can now publish the app on the public LiveChat Marketplace.
 
-- **Published** - App has been published to the public Marketplace. Developer can cancel the publication, or fork the application to make changes, and submit it new version to the App Review process.
+- **Published** - The app has been published to the public Marketplace. You can cancel the publication and we will take down your app from the LiveChat Marketplace. If you want to change your app, you need to fork it, and then submit it a new version for the App Review process once again.
 
 
 
 # Guidelines & Rules
 
-We are constantly improving App Review process to make it smooth for both you and us. For that purpose we have prepred few guidelines and rules that should make it easier.
+We are constantly improving the App Review process so that the cooperation is smooth for both parts. For that purpose, we have prepred a few guidelines and rules to make things go easier.
 
 ## Three bugs rule
 
-We stop reviewing the submitted app when we find 3 issues/bugs in it. After that, the app is rejected. The reason of rejection should be passed to the Developer via email.
+Once we find 3 bugs or issues in your app, we discontinue the review and reject the app. We will share with you the reason of rejection via email. 
 
-## Make a Tutorial
+## Make a tutorial
 
-Every app should have a tutorial that will cover the basic usage and that will explain all its features. Example: https://www.livechatinc.com/help/chatbot-integration/
+Every app should have a tutorial that covers its basic usage and explains all its features. As an example, see how the [ChatBot integration](https://www.livechatinc.com/help/chatbot-integration/) is described.
 
 ## Design system
 
-We recommend using our Design System to keep Apps consistent with the look & feel of the LiveChat application: https://developers.livechatinc.com/docs/design-system/
+We recommend using our Design System to keep apps consistent with the LiveChat look & feel. 
 
+<SectionLink to={"/design-system/"}>
+  See Design System
+</SectionLink>
 
 # FAQ
```

Comment:
What about this section? I would propose getting rid of it until we actually get some questions. "Coming soon" doesn't look good. Or maybe we already have some recurring questions from the past?